imports/plugins/core/layout/client/components/footer.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
Adding a footer requires design direction from @rymorgan since for the longest time, we've not had one displayed. And the question is, should we even be adding visible one now.

Reply:
Footer should not be displayed anywhere, if it wasn't already before..
Right now it's nothing more than an exemplary placeholder that's available for usage when implementing plugins.

Reply:
But then, a plugin would have its own requirements on how the Footer is supposed to be rendered. Maybe it's just unecessary to have it in its own place.